crates/primitives/Cargo.toml

```diff
@@ -34,6 +34,7 @@ sp-core = { workspace = true }
 
 sp-io = { version = "40.0.0", default-features = false, features = ["disable_panic_handler", "disable_oom", "disable_allocator"] }
 sp-runtime-interface = { version = "29.0.1", default-features = false, features = ["disable_target_static_assertions"] }
+sp-weights.workspace = true
```

Comment:
Seemed tolerable to add a dependency based on other existing sp_* usages. Whitespace changes are auto-formatting, mostly tabs being converted to 4 spaces.